deploy/charts/harvester/crds/harvesterhci.io_users.yaml

```diff
@@ -1,71 +0,0 @@
-
```

Comment:
I am not sure what is this CRD for because I have never seen this resource since I joined Harvester project. Do you have any idea about this? Thank you! @guangbochen @gitlawr

Reply:
The `users.harvesterhci.io` CRD is a leftover that we missed to clean up, currently we are using rancher users only.

Reply:
FYI https://github.com/harvester/harvester/pull/959